drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -32,13 +32,30 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
     """
     XBlock that implements a friendly Drag-and-Drop problem
     """
+    STANDARD_MODE = "standard"
+    ASSESSMENT_MODE = "assessment"
+
     display_name = String(
         display_name=_("Title"),
         help=_("The title of the drag and drop problem. The title is displayed to learners."),
         scope=Scope.settings,
         default=_("Drag and Drop"),
     )
 
+    mode = String(
+        display_name=_("Mode"),
+        help=_(
+            "Standard mode: feedback is provided to learner right after an item is dropped to a zone. "
```

Comment:
Yes, I'm covering DnDv2.

"Standard mode: the problem provides immediate feedback each time a learner drops an item on a target zone."

"Assessment mode: the problem provides feedback only after a learner drops all available items on target zones."

I can go through the rest of the file, when do you need that?